src/main/java/org/folio/search/configuration/properties/FolioEnvironment.java

```diff
@@ -10,10 +14,20 @@
   private final String okapiUrl;
   private final String env;
 
-  public FolioEnvironment(@Value("${okapi.url}") String okapiUrl,
-    @Value("${env:folio}") String env) {
-
+  public FolioEnvironment(@Value("${okapi.url}") String okapiUrl) {
     this.okapiUrl = okapiUrl;
-    this.env = env;
+    this.env = getFolioEnvName();
+  }
+
+  public static String getFolioEnvName() {
+    return validateFolioEnv(firstNonBlank(getenv("ENV"), getProperty("env"), "folio"));
+  }
```

Comment:
Why is this handled different from other docker environment variables?

Reply:
Because logic that builds index name is static (see `org.folio.search.utils.SearchUtils#getElasticsearchIndexName(java.lang.String, java.lang.String)`), it is the easiest way to implement it without refactoring to be non-static.